### PR TITLE
Docs/ACL

### PR DESCRIPTION
**Pull request by carpawell**

### CODE REVIEW

File: acl/doc.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	│  │  │  │  │ RANGEHASH │   RANGE   │   SEARCH  │ <- Object service method
+	│  │  │  │  ├──┬──┬──┬──┼──┬──┬──┬──┼──┬──┬──┬──┤
+	│  │  │ X│ F│ U│ S│ O│ B│ U│ S│ O│ B│ U│ S│ O│ B│ <- Rule
+	├──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┼──┤
+	│15│14│13│12│11│10│09│08│07│06│05│04│03│02│01│00│ <- Bit
+	├──┴──┴──┴──┼──┴──┴──┴──┼──┴──┴──┴──┼──┴──┴──┴──┤
+	│   DELETE  │    PUT    │   HEAD    │    GET    │ <- Object service method
+	├──┬──┬──┬──┼──┬──┬──┬──┼──┬──┬──┬──┼──┬──┬──┬──┤
+	│ U│ S│ O│ B│ U│ S│ O│ B│ U│ S│ O│ B│ U│ S│ O│ B│ <- Rule
+	└──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┴──┘
```
Comment (fyrchik) [inline acl/doc.go:29]: Nice table! Could you also add something about the first 2 bits? (should they be set to 0 or they are ignored, etc.).

**Review by fyrchik — APPROVED** [review]

File: acl/doc.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*
+Package acl provides primitives to perform handling basic ACL management in NeoFS.
+
+BasicACL type provides functionality for managing container basic access-control list.
+For example, setting public basic ACL that could not be extended with any eACL rules:
+
+import "github.com/nspcc-dev/neofs-sdk-go/container"
+...
```
Comment (alexvanin) [inline acl/doc.go:8]: Need `\t` there.

image

**Comment by alexvanin** [comment]
I propose to use `[#170]` in commit message to link with original #170.

File: acl/doc.go (hunk 1 above)
Comment (carpawell) [reply acl/doc.go:29]: > Nice table!

all admiration to @alexvanin :)

> Could you also add something about the first 2 bits?

you mean something more than "Remaining bits are reserved and are not used." after the table?

**Review by alexvanin — APPROVED** [review]

**Pull request merged**